inline-java.cabal

```diff
@@ -21,8 +21,16 @@ source-repository head
   type:     git
   location: https://github.com/tweag/inline-java
 
+flag ghc-head
```

Comment:
Using `impl(ghc > 8.9)` instead of a flag, would probably cause the package to not build when ghc-8.9 is finally released.

The flag makes it more evident that the code we have here is only a transient solution.